website/docs/audit.md

```diff
@@ -3,7 +3,7 @@ id: audit
 title: Audit
 ---
 
-The audit functionality enables periodic evaluations of replicated resources against the policies enforced in the cluster to detect pre-existing misconfigurations. Audit results are stored as violations listed in the `status` field of the failed constraint.
+The audit functionality enables periodic evaluations of replicated resources against the policies enforced in the cluster to detect pre-existing misconfigurations. Audit results are stored as violations listed in the `status` field of the failed constraint and logged to stdout of the audit pod.
```

Comment:
Do we still want to say `replicated resources` since we are not replicating them to opa cache anymore?

Reply:
Good catch. Doing a more major refactor of the audit docs. Hopefully this scrubs stale information and makes them a bit more clear.